src/main/webapp/app/drug/drug.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, EventEmitter, Input, OnInit } from '@angular/core';
+import { TrialService } from '../service/trial.service';
+import { Drug, NcitDrug } from '../drug/drug.model';
+import { debounceTime, switchMap } from 'rxjs/operators';
+
+@Component({
+    selector: 'jhi-drug',
+    templateUrl: './drug.component.html',
+    styleUrls: ['drug.scss']
+})
+
+export class DrugComponent implements OnInit {
+    @Input() armInput = {};
+    drugsOptionsLoading = false;
+    selectedDrugs: Array<any> = [];
```

Comment:
Why not model the type of selectedDrugs?

Reply:
I modeled on it. This is used in drug.component.html `[(ngModel)]="selectedDrugs"`.